bears/planemo/PlanemoLintBear.py

```diff
@@ -0,0 +1,30 @@
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.PipRequirement import PipRequirement
```

Comment:
We prefer the parenthesis like below... 
`from dependency_management.requirements.PipRequirement import  (     PipRequirement)`